gandalf.js

```diff
@@ -1,7 +1,6 @@
 /**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.